api/envoy/config/listener/v2/api_listener.proto

```diff
@@ -8,13 +8,12 @@ option java_package = "io.envoyproxy.envoy.config.listener.v2";
 option java_outer_classname = "ApiListenerProto";
 option java_multiple_files = true;
 
-// [#not-implemented-hide:]
 // Describes a type of API listener, which is used in non-proxy clients. The type of API
 // exposed to the non-proxy application depends on the type of API listener.
 message ApiListener {
   // The type in this field determines the type of API listener. At present, the following
   // types are supported:
-  //   envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager (HTTP)
+  // envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager (HTTP)
```

Comment:
@htuch do you have any thoughts on how this type of documentation could be automatically kept up to date in the future? I'm wondering if we could somehow if a protodoc insertion point that inserts all known variants of a message or something like that. Not sure what the best thing to do here is. This will need to be kept up to date for v3 very soon for example.

Reply:
The RST anchors are automagically transformed via `protoxform`. I think we should add an annotation to support this, so that `protoxform` can (1) do consistency checking to ensure there aren't typos and (2) automatic upgrade.